ui/intermittents-view/constants.js

```diff
@@ -0,0 +1,37 @@
+import * as d3 from "d3";
```

Comment:
For improved dead code removal during the build, could you try:
```js
import { curveLinear } from 'd3';

// ...
  interpolate: curveLinear
```

...and see if that still works?